api/src/main/java/io/minio/messages/QueueConfiguration.java

```diff
@@ -94,6 +94,14 @@ public void setEvents(List<EventType> events) {
   }
 
 
+
+  /**
+   * Sets filter.
```

Comment:
With the same logic, and to be consistent, we might also want to change the get filter method:
```
/**
    * Returns filter.
*/
public Filter filter() {
    return filter;
}
```
 =>
```
/**
    * Returns filter.
*/
public Filter getFilter() {
    return filter;
}
```